### ISSUE TITLE
Configure Greenkeeper to exclude some libraries from automatic-upgrades

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-ember#481]
# Problem

Greenkeeper is making a lot of PRs and that's great, but at the very least, we should skip updating any libraries that are part of Ember-CLI. Those are updated alongside ember-cli, so there should be no reason to do so ahead of time.

https://github.com/greenkeeperio/greenkeeper#repository-options

- [ ] Identify libraries to exclude
- [ ] Exclude libraries
- [ ] Sign off that it works as expected with follow-up issue

**Comment by JoshSmith** [comment]
Per that link, ignored libraries can be defined in `package.json`.

**Comment by QzSG** [comment]
> we should skip updating any libraries that are part of Ember-CLI
You mean those dependencies used by ember-cli?
Or also the addon modules like ember-cli-deploy and the others?

**Comment by JoshSmith** [comment]
@QzSG the deps used by `ember-cli` I believe. Those usually get updated when it does, so avoids a mad rush of open PRs that could be resolved by one.

**Comment by QzSG** [comment]
@JoshSmith In that case theoretically the only libary that needs to be added to the greenkeeper ignore list is `ember-cli`.

which means that the following added to `package.json` should do it
```
"greenkeeper": {
    "ignore": [
      "ember-cli"      
    ]
  }
```

However, I do not have any idea how we can test it unless one of the inner dependencies inside `ember-cli` is updated. When that happens, it should be working as intended if GreenKeeper does not file a pull request for the updated internal dependencies.

**Comment by JoshSmith** [comment]
Realistically I don't think there's any way to test this. More just like walking in artillery rounds. Too far? Aim lower.

If we get too many PRs in next release we'll adjust and try again.

**Comment by QzSG** [comment]
@JoshSmith CircleCI Test has passed, kindly refer to above referenced pull request.
I will take note of any future dependecy updates in ember-cli and update accordingly to see if Greenkeeper ignores them.

**Comment by JoshSmith** [comment]
Closed due to conversation in #537.
